earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp

```diff
@@ -44,19 +44,16 @@ AssetVersionImplD::StateChangeNotifier::GetNotifier(
 }
 
 void
-AssetVersionImplD::StateChangeNotifier::AddParentsToNotify(const std::vector<std::string> & parents) {
+AssetVersionImplD::StateChangeNotifier::AddParentsToNotify(const std::vector<SharedString> & parents) {
+   //std::copy(parents.begin(), parents.end(), std::inserter(parentsToNotify, parentsToNotify.end()));
   for (const auto& p : parents)
     parentsToNotify.insert(p);
-  if (!MiscConfig::Instance().ConsolidateListenerNotifications) {
-    // If the user has opted not to consolidate notifications, send
-    // notifications immediately
     NotifyParents(nullptr);
```

Comment:
This line should be removed. It was only called if someone disabled the consolidate listener notifications optimization, but that is the default behavior now. With the current code we don't want to notify parents right away - we want to wait until we know all the parents to notify.